sdk/logical/logical.go

```diff
@@ -159,6 +159,9 @@ type Paths struct {
 	// On standby nodes, like all storage write operations, this will trigger
 	// an ErrReadOnly return.
 	WriteForwardedStorage []string
+
+	// RFC5785
+	WellKnown map[string]string
```

Comment:
unused? Also I'm not seeing the hook for disabling a mount removing it's entries from the redirect map?